examples/__testfixtures__/styled-components-to-ucl/interpolated.output.ts

```diff
@@ -1,13 +1,19 @@
 import { Box } from '@rbilabs/universal-components';
 
 const WIDTH_PX = 12;
-const HEIGHT_PX = 12;
+const HEIGHT_REM = 2;
+const PADDING_LEFT_PX = 4;
 
 export const InterpolatedVariables = Box.withConfig({
   width: `${WIDTH_PX}px`,
-  height: `pre${HEIGHT_PX}rem`,
-  paddingX: '$4',
-  paddingY: '$40',
+
+  // TODO: RN - unsupported CSS
+  // rem are not supported in the interpolated value: __1substitution__rem
```

Comment:
Ideally this would be the actual value, instead of `__1substitution__rem`, but not sure if it's worth the error.

The alternative would be to throw an error

Reply:
Actually, this value can be removed since its on the next line